src/Homework.java

```diff
@@ -0,0 +1,50 @@
+public class Homework {
+
+    public static void main(String[] args) {
+        int num1 = -5;
+        int num2 = 5;
+        String hello = "Привет, Олег!";
+        int year = 2021;
+
+        System.out.println(sumTrueFalse(num1, num2));
+        numberPositivePrint(num1);
+        System.out.println(numberNegativeTrue(num1));
+        stringRepeat(num2, hello);
+        System.out.println(leapYear(year));
+    }
+
+    private static boolean sumTrueFalse(int a, int b) {
+        if ((a + b) >= 10 && (a + b) <= 20) {
+            return true;
+        }
+        return false;
```

Comment:
поскольку логическое выражение само по себе возвращает boolean результат, то можно упростить:
```suggestion
        return (a + b) >= 10 && (a + b) <= 20;
```